buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1358,6 +1363,14 @@ def set_build_extra_options(build_specs=null) {
         }
 
         EXTRA_MAKE_OPTIONS = params.EXTRA_MAKE_OPTIONS
+        if (params.CODE_COVERAGE) {
+            def CODE_COVERAGE_MAKE_OPTIONS = "EXTRA_CMAKE_ARGS=-DCODE_COVERAGE=ON"
+            if (!EXTRA_MAKE_OPTIONS) {
+                EXTRA_MAKE_OPTIONS = CODE_COVERAGE_MAKE_OPTIONS
+            } else {
+                EXTRA_MAKE_OPTIONS += " " + CODE_COVERAGE_MAKE_OPTIONS
+            }
+        }
         if (!EXTRA_MAKE_OPTIONS) {
             EXTRA_MAKE_OPTIONS = buildspec.getVectorField("extra_make_options", SDK_VERSION).join(" ")
```

Comment:
I'm not sure enabling code coverage should disable this.

Reply:
`EXTRA_MAKE_OPTIONS` in `CODE_COVERAGE` has been modified to combine default specs. Thanks.
New test links:
- xlinux, plinux, zlinux: Internal_J/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/10415/
- aarch64:  Internal_J/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/10416/

Reply:
I think this is still a problem. If I specify `CODE_COVERAGE`, then `EXTRA_MAKE_OPTIONS` will have a value and `SDK_VERSION` will not be considered.

Reply:
Sorry I did not notice the change did not refresh in this display. For new modification, I have moved CODE_COVERAGE related implementation below this code to avoid affect it. Just noticed the change is still not showing here, it can be reviewed in "file changed" section. Thanks.
New test link: Internal_Jenkins/job/Build_JDK11_ppc64le_linux_Personal/1121/